usagov/vote-gov-drupal
# VOTE-2016 Add image field to state and voter guide content types

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2016

## Description

Add image field to state and voter guide content types

## Deployment and testing

### Post-deploy steps

1. run `lando retune`

### QA/Testing instructions

1. Visit http://vote-gov.lndo.site/node/7/edit and add an image to the state page
2. Confirm that the image displays on the state page.
3. Visit http://vote-gov.lndo.site/node/82/edit and add an image to the voter guide
4. Confirm that the image displays on the voter guide page.

## Checklist for the Developer




## Checklist for the Peer Reviewers
